public/index.html

```diff
@@ -12,7 +12,9 @@
 
     <script>
       const protocol = window.location.protocol;
-      window.EXPRESS_PUBLIC_URL = `${protocol}//localhost:3000`;
+      const port = protocol === 'https:' ? 4000 : 3000;
+
+      window.EXPRESS_PUBLIC_URL = `${protocol}//localhost:${port}`;
       window.REACT_PUBLIC_URL = `${protocol}//localhost:3001`;
```

Comment:
@dompuiu does this need to be updated as well?

Reply:
@brenthosie I am not sure if the create-react-app offers an option to run the dev server on 2 different ports at the same time. I am not even sure if we use this variable anywhere. I am not able to find it.

Reply:
if it's not there, let's pull it out on this PR.

Reply:
I've removed it.